person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative('./nameable')
 
 class Person < Nameable
```

Comment:
- [ ] A method for adding rentals should be present in the `Person` class, kindly do so. Check the example below

e.g
```
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```